docs/redis.md

```diff
@@ -1,3 +1,71 @@
+# `redis` Metrics Receiver
+
+The redis receiver can retrieve stats from your redis server through the [INFO](https://redis.io/commands/info) command. 
+
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your redis configuration.
+
+To configure a receiver for your redis metrics, specify the following fields:
+
+| Field                 | Default                   | Description |
+| ---                   | ---                       | ---         |
+| `type`                | required                  | Must be `redis`. |
+| `endpoint`            | `localhost:6379`          | The url exposed by redis |
+| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+| `password`            |                           | The password used to connect to the server. |
+
+Example Configuration:
+
+```yaml
+metrics:
+  receivers:
+    redis_metrics:
+      type: redis
+      endpoint: localhost:6379
+      collection_interval: 30s
+      password: pwd
+  service:
+    pipelines:
+      redis_pipeline:
+        receivers:
+          - redis_metrics
+```
+
+## Metrics
+
+The Ops Agent collects the following metrics from your redis servers.
+
+| Metric                                                              | Data Type | Unit    | Labels  | Description    |
+| ---                                                                 | ---    | ---     | ---     | ---            | 
+| workload.googleapis.com/redis/uptime	                              | sum	   | s		   |         | Number of seconds since Redis server start |
```

Comment:
Why are all of these metrics slash-separated and not dot-separated? The scoping doc appears to call this "redis.uptime", though it also uses slashes for the subdirectories below.

Reply:
Can we add some translation to keep the separator consistent?

Reply:
I think this raises a larger question of whether or not metric naming is going to be deferred to the OTel community, and if not, to what extent should naming be "corrected" at the ops agent level?

Reply:
> Can we add some translation to keep the separator consistent?

for this one at least, I've opened a PR against the contrib repo to update the separator to be more consistent with the existing metrics.

Reply:
Current status: This is being changed in upstream PR: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5837 (Update Redis receiver's metric names)

Reply:
That's merged now - can you update this PR?

Reply:
updated

Reply:
Thanks for updating it!

Reply:
I think this doc still needs to be updated with the new metric names, and to remove the uptime metric.

Reply:
Nice catch. @Mrod1598 - Can you help update the `docs/redis.md` file?

Reply:
updated